app/CL_Editor/src/main/java/org/jjazz/cl_editor/barbox/api/BarBox.java

```diff
@@ -438,6 +460,8 @@ public void moveItem(ChordLeadSheetItem<?> item)
                 br.moveItemRenderer(item);
             }
         }
+        // TODO #534 Is this triggered when moving bars in CLS that are in the easy read "window"?
+        transposeItemRenderers();
```

Comment:
not needed. This move is **within** the same bar. BarRenderer.moveItem() just calls revalidate().   Moving an item from one bar to another is handled by CL_EditorImpl which removes item from one BarBox and add it to another one.